src/Routers/FunctionsRouter.js

```diff
@@ -212,6 +183,9 @@ export class FunctionsRouter extends PromiseRouter {
         }
       );
       return Promise.resolve()
+        .then(() => {
+          return triggers.maybeRunValidator(request, functionName);
+        })
         .then(() => {
           return theFunction(request, { message });
```

Comment:
I just realized something. `message` can be removed.

`Parse.Cloud.define('MyClass', (req, message)=> {})` is a thing.

Looks like it mixed up with the Jobs implementation.

Reply:
Needs to be remove from the `createResponseObject` function and return also (2 places);

Reply:
So (Line 102)

`static createResponseObject(resolve, reject, message)`

to

`static createResponseObject(resolve, reject)`

And line 116 to be removed?

Reply:
Yeah and the lint error

Reply:
Removing those lines results in tests failing:

```
1) Cloud Code should fully delete objects when using `unset` with beforeSave (regression test for #1840)
  Message:
    Failed: {"message":"Cannot read property 'error' of undefined","code":141}
  Stack:
    Error: Failed: {"message":"Cannot read property 'error' of undefined","code":141}
        at <Jasmine>
        at global.jfail (/parse-server/spec/helper.js:408:3)
        at /parse-server/spec/CloudCode.spec.js:1369:9
        at <Jasmine>
        at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

Reply:
Yeah those tests need to be updated. If you want I can do a separate PR.

Reply:
Sounds good if that's no stress mate. Otherwise I can try dig to the bottom of it if you're tied up.